Documentation/joss/paper.md

```diff
@@ -19,50 +9,85 @@ authors:
     orcid: 0000-0003-0387-3440
     affiliation: 1
   - name: Helge Hecht
-    affiliation: 2
+    affiliation: 1
   - name: Julian Amann
     affiliation: 1
   - name: Jonas Schlenger
+    orcid: 0000-0002-4875-5117
     affiliation: 1
```

Comment:
one author is missing here

Reply:
https://joss.readthedocs.io/en/latest/submitting.html#authorship

I asked him, waiting for a response.